src/components/utilities/requests.jsx

```diff
@@ -16,9 +16,16 @@ export const fetchAllArticles = async (topic) => {
 export const fetchArticleById = async (article_id) => {
   const { data } = await baseURL.get(`/articles/${article_id}`);
   return data.article;
```

Comment:
If you're using async/await you need to have your code in a try/catch block to handle errors:

`try {
// code in here
} catch(err){
// Error handling
}`